code/modules/clothing/head/head.dm

```diff
@@ -283,6 +306,29 @@
 		"unflipped" = "You hook the hat's chinstrap under your chin. Peace of mind is worth a little embarassment."
 		)
 
+/obj/item/clothing/head/hood/
+	name = "\improper USCM hood"
+	desc = "A hood issued to USCM marines. Often worn by scouts and snipers navigating harsh conditions. Protects you from the weather, if nothing else"
+	icon_state = "hood"
+	var/list/dropping_message = list(
+		"dropped" = "You drop the hood down, revealing your face.",
+		"raised" = "You raise the hood up, concealing your face."
+		)
```

Comment:
very shitcode, just make 2 separate vars

Reply:
I'm a completely inept coder; just a spriter with some *very basic* literacy. How would you reccommend I redo this?

Reply:
I think I fixed it. Resolving for now.